raiden/tests/integration/test_tokenswap.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+import pytest
+import gevent
+
+from raiden.tests.utils.transfer import (
+    assert_synched_channels,
+    channel,
+)
+
+# Use a large enough settle timeout to have valid transfer messages
+TEST_TOKEN_SWAP_SETTLE_TIMEOUT = (
+    5 +  # reveal timeout
+    7 +  # maker expiration
+    7    # taker expiration
+)
+
+@pytest.mark.timeout(160)
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [2])
```

Comment:
wouldn't it make sense to also have the actual mediated scenarios with `Maker`, `Bob`, `Charly` and `Taker`?
M -> B -> T
T -> C -> M